understanding/parsing.html

```diff
@@ -55,92 +53,75 @@
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Assistive technology can properly present page content.</dd>
+                  <dt>目標</dt>
+                  <dd>支援技術はページコンテンツを適切に表示できる。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Create web pages according to specifications.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>仕様に従ってウェブページを作成する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People can browse web content more easily with their assistive technology.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>支援技術を使用すると、人々はウェブコンテンツをより簡単に閲覧できる。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <div class="wcag22"> <span class="new-version">New in WCAG 22: </span>
+               <h2>意図</h2>
+               <div class="wcag22"> <span class="new-version">WCAG 2.2 の新基準:</span>
                   
-                  <p>This criterion has been removed from WCAG 2.2.</p>
+                  <p>この基準は WCAG 2.2 から削除された。</p>
                   
                   
-                  <p>The intent of this Success Criterion was to ensure that user-agents, including assistive
-                     technologies, can accurately interpret and parse content. Since WCAG 2.0 was published,
-                     the specifications (such as HTML) and browsers have improved their handling of parsing
-                     errors. It is also the case that assistive technology used to do their own parsing
-                     of markup, but now rely on the browser. For that reason this success criterion has
-                     been removed. Many issues that would have failed this criterion will fail <a href="info-and-relationships">Info and Relationships</a> or <a href="name-role-value">Name, Role, Value</a>. Other issues are excepted by the "except where the specification allow these features"
-                     part of the criterion.
+                  <p>この達成基準の意図は、支援技術を含むユーザエージェントがコンテンツを正確に解釈及び解析できるようにすることであった。WCAG 2.0 が公開されて以来、仕様 (HTML など) 及びブラウザは解析エラーの処理方法を改善してきた。また、支援技術はかつては独自にマークアップの解析を行っていたが、現在はブラウザに依存している。そのため、この達成基準は削除された。この基準に失敗した多くの問題は、<a href="info-and-relationships">情報及び関係性</a>又は、<a href="name-role-value">名前 (name)・役割 (role)・値 (value)</a> に失敗する。その他の問題は、基準の「ただし、仕様で認められているものを除く」という部分によって除外される。
                   </p>
                   
                   
-                  <p>The following content is left for historical purposes to show the original intent.</p>  
+                  <p>次の内容は、元の意図を示すために歴史的な目的で残されている。</p>  
```

Comment:
「歴史的」というと大仰な感じがするのですが、ここでの history は履歴くらいの意味合いでしょうか。

```suggestion
                  <p>次の内容は、元の意図を示すための履歴の目的で残されている。</p>  
```

Reply:
履歴というのもちょっと違うような…？
大きな問題がなければ、ママとしてもらえればと。